libs/fileset/file.go

```diff
@@ -18,3 +22,16 @@ func (f File) Modified() (ts time.Time) {
 	}
 	return info.ModTime()
 }
+
+func (f File) WsfsPath() (string, error) {
```

Comment:
nit: I'd call it RemoteRelativePath instead

Reply:
Well, remote also includes DBFS and Files API. They don't have the property of stripping the file extension based on the file content.

Reply:
Ended up removing the function all together because we need to ignore errors during `notebook.Detect` (to maintain backwards compatibility)